pkg/aws/client.go

```diff
@@ -431,10 +431,21 @@ func (c *awsClient) GetRegion() string {
 	return aws.StringValue(c.awsSession.Config.Region)
 }
 
-func (c *awsClient) GetSubnetIDs() ([]*ec2.Subnet, error) {
-	return c.getSubnetIDs(&ec2.DescribeSubnetsInput{})
-}
+func (c *awsClient) ListSubnets(subnetIds ...string) ([]*ec2.Subnet, error) {
+
+	if subnetIds == nil || len(subnetIds) == 0 {
+		return c.getSubnetIDs(&ec2.DescribeSubnetsInput{})
+	}
```

Comment:
No need to check for `nil`
```suggestion
	if len(subnetIds) == 0 {
		return c.getSubnetIDs(&ec2.DescribeSubnetsInput{})
	}
```